controller/coq_lsp.ml

```diff
@@ -44,7 +44,7 @@ let rec process_queue ~ofn ~state =
        I/O *)
     exit 0
   | Some (Yield state) ->
-    Thread.delay 0.1;
+    Thread.delay 0.01;
```

Comment:
Why do we delay at all actually?

Reply:
Errrm, that's actually a pretty good question, I'm not sure. @ejgallego maybe you can answer that?

Reply:
Otherwise when coq-lsp is idle we would eat all of the CPU busy-waiting.

We could use thread-aware queues, so the thread could be put to sleep and woken up by that, **however** that may work badly on Windows and jsCoq so I prefer this very simple method until our testing infra is better on these platforms.

Happy if someone wants to do an experiment with https://v2.ocaml.org/releases/4.14/api/Condition.html

Reply:
But yes, that means with our setup we often get a 0.1 latency, as I wrote on the other comment we better make this a parameter and find the right spot (UI programming .... XD)